tests/php/TemplatingTest.php

```diff
@@ -6,6 +6,9 @@
 use PHPUnit_Framework_TestCase;
 use WMDE\VueJsTemplating\Templating;
 
+/**
+ * @license GPL-2.0-or-later
```

Comment:
is GPL 2.0 or later the same thing as LGPL 2.1 this library is licensed under?

Reply:
No, that’s a pretty big difference…